lib/jsdom/living/postmessage.js

```diff
@@ -0,0 +1,25 @@
+"use strict";
+const isValidTargetOrigin = require("../utils").isValidTargetOrigin;
+
+module.exports = function (message, targetOrigin) {
+  let window = this;
+  let event;
+
+  if (!targetOrigin) {
+    throw new TypeError("'postMessage' requires a second argument 'targetOrigin'");
+  }
+
+  if (!isValidTargetOrigin(targetOrigin)) {
+    throw new SyntaxError("Failed to execute 'postMessage' on 'Window': " +
```

Comment:
Is the correct invocation `throw new DomException("SyntaxError", "Failed to execute 'postMessage'...`?